packages/runtime/container-runtime/src/id-compressor/identifiers.ts

```diff
@@ -0,0 +1,53 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { OpSpaceCompressedId, SessionSpaceCompressedId } from "@fluidframework/runtime-definitions";
+
+/**
+ * An identifier (v4 UUID) that has been shortened by a distributed compression algorithm.
+ * Lacks a space (session/op), meaning its scope is the same as the space-specific ID from which it was derived.
+ * @alpha
+ */
+export type CompressedId =
```

Comment:
sadly, it can't be done due to the only usages needing to pass IDs of single types (not combinations). I was able to remove the CompressedId type, though.